.buildkite/common.py

```diff
@@ -29,14 +31,15 @@ def group(label, command, instances, kernels, agent_tags=None, **kwargs):
     label1 = label[0]
     steps = []
     for instance in instances:
-        for kv in kernels:
-            agents = [f"instance={instance}", f"kv={kv}"] + agent_tags
-            step = {
-                "command": command,
-                "label": f"{label1} {instance} {kv}",
-                "agents": agents,
-                **kwargs,
-            }
-            steps.append(step)
+        for os in distros:
+            for kv in kernels:
```

Comment:
I think the os/kernel should go together; we will not run al2023 with 4.14. so we probably want sthg like a list of tuples:

```python
DEFAULT_PLATFORMS = [
  (al2, 4.14),
  (al2, 5.10),
  (al2023, 6.1),
]
```

But for now we can maybe just add `os=al2` in `pipeline_pr.py:94`, and that should apply to the whole pipeline?

Reply:
Should be fixed now.